subprojects/docs/src/docs/userguide/commandLineInterface.adoc

```diff
@@ -0,0 +1,520 @@
+----
+gradle taskName [--options]
+----
+
+Options that accept values can be specified with or without `=` between the option and argument. The following are equivalent.
+----
+--console=plain
+--console plain
+----
+
+Options that enable behavior have long-form options with inverses specified with `--no-`. The following are opposites.
+----
+--build-cache
+--no-build-cache
+----
+
+Many long-form options, have short option equivalents. The following are equivalent:
+----
+--help
+-h
+----
+
+Many command-line flags can be specified in `gradle.properties` to avoid needing to be typed. See the <<sec:gradle_properties_and_system_properties, configuring build environment guide>> for details.
+
+The following sections describe use of the Gradle command-line interface, grouped roughly by user goal.
+
+[[sec:command_line_executing_tasks]]
+=== Executing Tasks
+
+You can run a task and all of its dependencies
+----
+❯ gradle myTask
+----
+
+You can learn about what projects and tasks are available in the <<sec:command_line_project_reporting, project reporting section>>.
+
+==== Executing tasks in multi-project builds
+In a <<intro_multi_project_builds, multi-project build>>, subproject tasks can be executed with ":" separating subproject name and task name. The following are equivalent _when run from the root project_.
+
+----
+❯ gradle :mySubproject:taskName
+❯ gradle mySubproject:taskName
+----
+
+You can also run a task for all subprojects using the task name only. For example, this will run the "test" task for all subprojects when invoked from the root project directory.
+
+----
+❯ gradle test
+----
+
+When invoking Gradle from within a subproject, the project name should be omitted:
+
+----
+❯ cd mySubproject
+❯ gradle taskName
+----
+
+==== Executing multiple tasks
+You can also specify multiple tasks. For example, the following will execute the `compile` and `test` tasks in the order that they are listed on the command-line and will also execute the dependencies for each task.
+
+----
+❯ gradle compile test
+----
+
+[[sec:excluding_tasks_from_the_command_line]]
+==== Excluding tasks from execution
+You can exclude a task from being executed using the `-x` or `--exclude-task` command-line option and providing the name of the task to exclude.
+
+++++
+<figure>
+    <title>Example Task Graph</title>
+    <imageobject>
+        <imagedata fileref="img/commandLineTutorialTasks.png"/>
+    </imageobject>
+</figure>
+++++
+
+++++
+<sample id="excludeTask" dir="userguide/tutorial/excludeTasks" title="Excluding tasks">
+    <output args="dist --exclude-task test"/>
+</sample>
+++++
+
+You can see that the `test` task is not executed, even though it is a dependency of the `dist` task. The `test` task's dependencies such as `compileTest` are not executed either. Those dependencies of `test` that are required by another task, such as `compile`, are still executed.
```

Comment:
I'll see what I can do here. Seems like a build scan would be cool.

Reply:
I am not sure if a build scan would properly visualize it. I guess it would just say "SKIPPED". I was thinking more in terms of a graph here. Happy not to have it for now. It would just be helpful for better understand that task dependencies of excluded task are not excluded as well.